one-trust-notices/src/config/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const ONE_STRUST_API_ENDPOINT = 'https://app-de.onetrust.com/api'
+export const ONE_TRUST_NOTICES = [
+  {
+    name: 'Terms of service',
+    id: '6bf8412a-41a7-41a0-82dc-26286ce61b1a',
+  },
+  {
+    name: 'Privacy policy',
+    id: '0df21ff6-3e8f-4320-af8f-23dea9135d57',
```

Comment:
it would be nice if these two ids were taken from env vars

Reply:
Ups, I forgot! Thanks